Docs: Add info about logs/metrics unification to What's New docs (#26325)
grafana/grafana: docs/sources/guides/whats-new-in-v7-1.md

- [**Deep linking for Google Cloud Monitoring (formerly named Google Stackdriver) datasource**]({{< relref "#deep-linking-for-google-cloud-monitoring-formerly-named-google-stackdriver-datasource" >}})
- [**Transforms**]({{< relref "#transforms" >}})
- [**Stat panel text mode**]({{< relref "#stat-panel-text-mode" >}})

> - [**Unification of Explore modes**]({{< relref "#explore-mode-unification" >}})

Reviewer: ```suggestion
- [**Unification of Explore modes**]({{< relref "#explore-modes-unified" >}})
```

{{< docs-imagebox img="/img/docs/v71/stat-panel-text-modes.png" max-width="1025px" caption="Stat panel" >}}

> ## Unification of Explore modes

Reviewer: ```suggestion
## Explore modes unified
```

> New to Grafana is 7.1 is a fairly major change to Explore that does away with the query mode selector altogether.

Reviewer: ```suggestion
Grafana 7.1 includes a major change to Explore: it removes the query mode selector.
```

> Many data sources are in a position where they can tell us whether a response contains time series data or logs data, and using this information, Explore now chooses which visualization to use for that data. This means there’s no need to switch back and forth between Logs and Metrics modes depending on the type of query you want to make, leading to more productive exploration of your data. 

Reviewer: ```suggestion

Many data sources tell Grafana whether a response contains time series data or logs data. Using this information, Explore chooses which visualization to use for that data. This means that you don't need to switch back and forth between Logs and Metrics modes depending on the type of query that you want to make. 
```
